src/more-infos/more-info-media_player.js

```diff
@@ -136,6 +167,24 @@ export default new Polymer({
     this.callService('volume_mute', { is_volume_muted: !this.isMuted });
   },
 
+  handleVolumeUp() {
+    const obj = this.$.volumeUp;
+    this.handleVolumeWorker('volume_up', obj, true);
+  },
+
+  handleVolumeDown() {
+    const obj = this.$.volumeDown;
+    this.handleVolumeWorker('volume_down', obj, true);
+  },
+
+  handleVolumeWorker(service, obj, force) {
+    if (force || (obj !== undefined && obj.pointerDown)) {
```

Comment:
https://elements.polymer-project.org/elements/paper-icon-button#pointerDown

That is where I got it from.